src/time/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ add_library(Time STATIC
 )
 
 target_include_directories(Time
+        PUBLIC ${SDL_INCLUDE_DIR}
```

Comment:
Please create INTERFACE target carrying `SDL_INCLUDE_DIR` and `SDL_LIBRARY` for convenience.
Call it SDL_1_XX.